spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -218,12 +218,23 @@ def pack_measures(self):
         writer.save()
 
     def show_total_results(self):
+        """
+        Print total results to CLI
+        """
+
         printv('\n\nAveraged measures...', self.verbose, 'normal')
-        for stg, key in zip(['  Volume [mm^3] = ', '  (S-I) Length [mm] = ', '  Equivalent Diameter [mm] = '], ['volume [mm3]', 'length [mm]', 'max_equivalent_diameter [mm]']):
-            printv(stg + str(np.round(np.mean(self.measure_pd[key]), 2)) + '+/-' + str(np.round(np.std(self.measure_pd[key]), 2)), self.verbose, type='info')
 
-        printv('\nTotal volume = ' + str(np.round(np.sum(self.measure_pd['volume [mm3]']), 2)) + ' mm^3', self.verbose, 'info')
-        printv('Lesion count = ' + str(len(self.measure_pd['volume [mm3]'].values)), self.verbose, 'info')
+        for label, key in zip(['Volume [mm^3]', '(S-I) Length [mm]', 'Equivalent Diameter [mm]'], self.measure_keys):
```

Comment:
We could maybe simplify things a little more by making `self.measure_keys` equivalent to the labels, so that all we have to do is iterate through `self.measure_keys` (acting as both the labels and keys)?

Reply:
Good idea! Simplified in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4207/commits/68683a37a939904149da14effa08f111da1283c0